client/components/form/Radio.tsx

```diff
@@ -18,10 +19,11 @@ export interface RadioType extends React.FC<RadioProps> {
   Boolean: React.FC<BooleanRadioProps>;
 }
 
-export const Radio: RadioType = ({ legend, name, options, horizontal }) => {
+export const Radio: RadioType = ({ legend, name, options, horizontal, description }) => {
   return (
     <fieldset className='flex flex-col gap-4'>
       <legend className='text-bcBlack font-bold mb-4'>{legend}</legend>
+      <Description id={`${name}-description`}>{description}</Description>
```

Comment:
I'm not sure that this description will be properly exposed as a part of the fieldset